dev/start

```diff
@@ -138,13 +138,23 @@ if ! [ ${CONDA_SHLVL:-0} = 0 ]; then
     done
 fi
 
+UNAME="$(uname)"
+MACHINE="$(uname -m)"
+UNAME_SM="$(uname -sm)"
+
+# $ uname -sm
+# Linux x86_64
+# Linux aarch64
+# Darwin arm64
+# ...
+
```

Comment:
This needs to be moved up before the arg parsing around line 33.

Also, since this script is sourced we've been very careful with cleaning up all temp vars, so let's prefix all of these with `_` and add to the cleanup function defined at the top.

Reply:
Done